metadata.rb

```diff
@@ -14,4 +14,4 @@
 
 depends          "apache2",          "= 3.2.2"
 depends          "perl",             "= 4.0.0"
-depends          "postfix",          "= 3.7.0"
+depends          "postfix",          "= 5.3.1" # same as in t3-base
```

Comment:
As we no longer want to have a specific postfix version maybe remove this line completely instead?

Reply:
I thought it would be good to keep the postfix dependency in this cookbook because that's essential for a system that sends mails. Suggestion: Add dependency on `postfix ~> 5.3.0` (or `postfix > 5.3.0)?

Reply:
Yes, I'd vote to allow 5.3.x releases at least.

Reply:
Done